data/routes.php

```diff
@@ -7,7 +7,7 @@
 /* Commonly used regular expressions */
 
 // Optional calendar short name, which prefixes all routes
-$calendar                = '(?P<calendar_shortname>([a-zA-Z-_]([0-9]+)?)+)';
+$calendar                = '(?P<calendar_shortname>([a-zA-Z-_0-9]+)+)';
```

Comment:
I'd move the `0-9` part after the `A-Z` part, keeping all of the symbols at the end of the character reference. That way we avoid confusion about when `-` is a literal and when it's a span delimiter.